src/main/java/me/zeroeightsix/kami/module/modules/combat/Aura.kt

```diff
@@ -40,7 +41,7 @@ class Aura : Module() {
     private val waitTick = register(Settings.floatBuilder("SpamDelay").withMinimum(0.1f).withValue(2.0f).withMaximum(40.0f).withVisibility { delayMode.value == WaitMode.SPAM }.build())
     private val eat = register(Settings.b("WhileEating", true))
     private val players = register(Settings.b("Players", true))
-    private val friends = register(Settings.booleanBuilder("Friends").withValue(false).withVisibility { players.value }.build())
+    private val friends = register(Settings.booleanBuilder("Friends").withValue(false).withVisibility { players.value && !KamiMod.MODULE_MANAGER.isModuleEnabled(AntiFriendHit::class.java) }.build())
```

Comment:
please revert this, there was no reason for this to be changed

Reply:
I just thought it would be a quality of life feature since if you cannot hurt your friends there is no reason to attempt to attack them. It would be a pain if when you enabled AntiFriendHit that you also had to disable the configuration in Aura. I'll revert it if you really want though, just tell me once and consider it done.

Reply:
It would just confuse users, as hiding the option when the option still makes a difference is dumb.

Reply:
It would not make a difference though, that is the point of the changes in EntityUtils. I will make a change or two and I want you to review it and see if it works better.

Reply:
I'd rather modules not affect this sort of thing. AntiFriendHit only exists to prevent manually left clicking on friends, that's it. It shouldn't be affecting other modules or utils.

Reply:
Okay, as an alternative can I just add a setting to AntiFriendHit and set the default to not affect other modules?

Reply:
It would make the code unnecessary complicated. I don't believe Antifriendhit should affect any other modules, they all have their own friend settings already.